doc/whats_new.rst

```diff
@@ -32,6 +32,10 @@ API
 - Connection `'src_gids'` and `'target_gids'` are now stored as set objects
   instead of lists, by `Ryan Thorpe`_ in :gh:`642`.
 
+- :func:`~hnn_core.Dipole.write` and :func:`~hnn_core.Dipole.read_dipoles`
+  now support hdf5 format for read/write Dipole object, by
+  `Rajat Partani`_ in :gh:`648`
```

Comment:
rebase problem?

Reply:
Yeah I think so! The commit is from awhile back so I think the best plan is to just focus on cleaning things up and we can walk through fixing the commit history over zoom or something

Reply:
This still needs to be fixed @carolinafernandezp. Perhaps try using [git-blame](https://www.git-scm.com/docs/git-blame) to identify which commit this was accidentally added on, and then edit it out with an interactive rebase.